test/e2e/user_workload_monitoring_test.go

```diff
@@ -1181,6 +1181,61 @@ func assertEnforcedTargetLimit(limit uint64) func(*testing.T) {
 	}
 }
 
+func assertNamespaceOptOut(t *testing.T) {
+	ctx := context.Background()
+
+	serviceMonitorJobName := "serviceMonitor/user-workload-test/prometheus-example-monitor/0"
+
+	// Ensure the target for the example ServiceMonitor exists.
+	f.ThanosQuerierClient.WaitForTargetsReturn(t, 5*time.Minute, func(body []byte) error {
+		return getActiveTarget(body, serviceMonitorJobName)
+	})
+
+	// Add opt-out label to namespace.
+	ns, err := f.KubeClient.CoreV1().Namespaces().Get(ctx, userWorkloadTestNs, metav1.GetOptions{})
+	if err != nil {
+		t.Fatalf("Failed to fetch user-workload namespace: %v", err)
+	}
+
+	labels := ns.GetLabels()
+	labels["openshift.io/user-monitoring"] = "false"
```

Comment:
Should we use a more 'explicit' label? `openshift.io/user-moitoring-excluded`?

Reply:
I understand that `true` means that you want to 'monitor' the namespace; and `false` indicates that you don't want to monitor; however not having the label does the same thing *(I think). Thus making sure this is clearly documented will be EXTREMELY important.

Reply:
I think the label was chosen to be consistent with the existing `openshift.io/cluster-monitoring` label. I don't have much of an opinion on the naming. I agree it needs to be documented. I will prep a docs PR.